libs/message-handler/package.json

```diff
@@ -4,19 +4,27 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "lint": "eslint '**/*.{js,ts,jsx,tsx}'",
     "dev": "vite",
-    "build": "tsc -b && vite build",
-    "lint": "eslint .",
+    "test": "",
+    "build": "tsc -b",
     "preview": "vite preview"
   },
   "dependencies": {
+    "@veramo/did-provider-peer": "^6.0.0",
+    "base64url": "^3.0.1",
     "didcomm": "^0.4.1",
+    "ethr-did-resolver": "^11.0.2",
+    "peer-did-resolver": "^2.0.0",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
-    "uuid": "^10.0.0"
+    "uuid": "^10.0.0",
+    "web-did-resolver": "^2.0.27"
```

Comment:
Make sure to remove the dependencies that are not needed for `message-handler` functionality.

Reply:
i prefer everything be removed for now and added only when needed